OrbitBase/include/OrbitBase/Logging.h

```diff
@@ -34,6 +35,8 @@
     PLATFORM_LOG(formatted_log__.c_str());                                    \
   } while (0)
 
+#define LOG_VAR(x) LOG("%s = %s", #x, std::to_string(x))
```

Comment:
Hide it behind ifdef DEBUG?

Reply:
No, I especially need it in release actually, it's kind of the point :-). It's really just syntactic sugar for the LOG macro...

Reply:
ok, this is something that us used locally but is not supposed to be actually commited to repo, right? If so I want to have some kind of protection against accidental uses of this instead of regular and more readable LOG messages in cases where LOG is intended and not for debug reasons.

Reply:
I agree. This is a debug utility more than a logging utility. That's why it was removed. We don't want usages of `LOG_VAR` to actually end up in the code base, or at least to not have an effect on logs in release, as the messages are quite cryptic. Of course one might still use it while developing but still building with optimizations, so I guess `#ifndef NDEBUG` is not the way to go, but this shouldn't be added as is to `Logging.h`.

Reply:
Guys, you are killing me. This has been in the code base for ages, anything it can do LOG can do as well, it's pure syntactic sugar.